### ISSUE TITLE
RSPeer landing page logo is not displaying properly.

### ISSUE THREAD

**Issue by Sphiinx** [MaddoxDevelopment/RSPeer-Api#12]
RSPeer landing page logo is not displaying properly. The images/client.png file is missing so it instead is displaying the alt text "phone".

**Comment by MaddoxDevelopment** [comment]
Got a screenshot? I'm not seeing this.

**Comment by Sphiinx** [comment]
This is fixed.
